### PR TITLE
code challenge 01

### PR DESCRIPTION
**Pull request by 0xQuasark**
- [X] Top-level README “Table of Contents” is updated
 - [X] README for this challenge is complete
       - [X] Summary, Description, Approach & Efficiency, Solution
       - [X] Picture of whiteboard
       - [X] Link to code
 - [n/a] Feature tasks for this challenge are completed
 - [n/a] Unit tests written and passing
       - [n/a] “Happy Path” - Expected outcome
       - [n/a] Expected failure
       - [n/a] Edge Case (if applicable/obvious)

### CODE REVIEW

File: 401 Challenges/challenge01/01.README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+- Images
+![Page1](image.png)
+![Page2](image-1.png)
+
+## Approach & Efficiency
+<!-- What approach did you take? Why? What is the Big O space/time for this approach? -->
+- I first paraphrased the question
+- Then I sketched out what would need to happen
+- Then I spoke about what needed to happen
+- I noticed the spec said not to use built in functions, so I didn't assume we'd know the length of the array
```
Comment (arpatterson31) [inline 401 Challenges/challenge01/01.README.md:16]: Great job Paul! The problem did say you couldn't use any built in methods but .length is considered a property so you could have certainly used it. But way to go figuring out how to get to the solution without even using the length!
